group_vars/bibdata/production.yml

```
@@ -151,7 +151,7 @@ datadog_config:
     analyzed_spans:
       bibdata|rack.request: 1
     filter_tags:
-      reject: ["http.useragent:nginx/1.23.4 (health check)", "operation:heartbeat", "operation:job_fetch", "operation:scheduled_push", "operation:scheduled_poller_wait"]
+      reject: ["http.useragent:nginx/1.23.4 (health check)", "operation:heartbeat", "operation:job_fetch", "operation:scheduled_push", "operation:scheduled_poller_wait", "resource_name:(sidekiq.heartbeat OR sidekiq.job_fetch OR sidekiq.scheduled_push)"]
```

```suggestion
      reject: ["http.useragent:nginx/1.23.4 (health check)", "operation:heartbeat", "operation:job_fetch", "operation:scheduled_push", "operation:scheduled_poller_wait"]
    ignore_resources: ["sidekiq.heartbeat", "sidekiq.job_fetch", "sidekiq.scheduled_push"]
```

Apparently, according to [the datadog docs](https://docs.datadoghq.com/tracing/guide/ignoring_apm_resources/?tab=datadogyaml#ignoring-based-on-resources), this type of configuration goes under `ignore_resources` rather than `filter_tags/reject`.